components/pagination/Pagination.js

```diff
@@ -0,0 +1,128 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import { LinkButton } from '../button';
+import { TextSmall } from '../typography';
+import { IconChevronLeftMediumOutline, IconChevronRightMediumOutline } from '@teamleader/ui-icons';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class Pagination extends PureComponent {
+  static propTypes = {
+    inverse: PropTypes.bool,
+    nextPageText: PropTypes.string,
+    numItems: PropTypes.number.isRequired,
+    numItemsPerPage: PropTypes.number,
+    maxNumPagesVisible: PropTypes.number,
+    currentPage: PropTypes.number.isRequired,
+    onPageChange: PropTypes.func.isRequired,
+    prevPageText: PropTypes.string,
+    className: PropTypes.string,
+  };
+
+  static defaultProps = {
+    currentPage: 1,
+    inverse: false,
+    numItemsPerPage: 10,
```

Comment:
I'm in doubt whether this component should know how many items there are on a page. Maybe we should just pass `numPages` instead?

Reply:
I think he wants to achieve the `show xx items`? If people want to render all items at once or not, then it will need to know this, right?

Reply:
Correct me if I'm wrong, but that's not a part of the pagination? It's not in current spec either.

Reply:
Hmm... Kind of a tough one? It does change the pagination, (more items per page decreases the amount of pages needed to show it) but indeed if you see it as a module (which you should), then pagination should be something split up. Okay I think it would be better to remove it from the Pagination component.

Also Q @lowiebenoot : If it's not in the spec but it will probably be implemented later, do you provide it or not?

Reply:
Another argument is that you might not know the amount of items. An API might just give you the amount of pages. It would be ridiculous to fake the amount of items by doing `numItemsPerPage * numPages`.

Reply:
If it's not in the spec, but it will probably be implemented later, you might take it into account already. But it's also not bad to not take it into account yet, and just refactor it later, when it is implemented. There's no right or wrong here, it just depends on the case.